apps/filecoin-site/src/app/_components/LogoSection/LogoSection.tsx

```diff
@@ -15,7 +15,7 @@ export function LogoSection({ logos, title, tag = 'h2' }: LogoSectionProps) {
       aria-labelledby={title ? 'logo-section-title' : undefined}
     >
       {title && (
-        <Heading tag={tag} variant="lg-regular" id="logo-section-title">
+        <Heading tag={tag} id="logo-section-title">
```

Comment:
I would prefer if we add a class and use that instead of the id. IDs are primarily meant for JavaScript targeting, page anchors, and form labels - not styling.

Reply:
In this case, however, we can just inline the styles.